components/Hero/Hero.tsx

```diff
@@ -179,7 +171,7 @@ export default function Hero() {
           }}
           className="pb-3 pt-3 sm:pt-4 md:pt-6 lg:pt-7 text-[12px] md:text-[20px] leading-[160%] sm:leading-[140%]"
         >
-          Anonymously prove any subset of any email <br></br>you sent or
+          Anonymously prove any subset of any email <span className="hidden sm:inline"><br /></span> you sent or
```

Comment:
same as above

Reply:
removing the line break on extra small screens but keeping it on screens small and above - for mobile responsiveness